discovery-infra/tests/test_validations.py

```diff
@@ -80,3 +81,30 @@ def test_host_insufficient(self, cluster, modified_nodes):
         new_cluster.wait_until_hosts_are_discovered(allow_insufficient=True)
         new_cluster.wait_for_host_validation(vcpu_host_id, "hardware", "has-min-cpu-cores", ["success"])
         new_cluster.wait_for_host_validation(ram_host_id, "hardware", "has-min-memory", ["success"])
+
+    @pytest.mark.regression
+    @pytest.mark.skip(reason="BZ:1897916")
+    def test_hosts_and_cluster_max_length(self, cluster, nodes):
+        # Trying to create a cluster with more then 54 chars
```

Comment:
Sometimes you mention 64 chars and sometimes 54 chars in the comments.
I would prefer to generate a new `cluster_name` each time with the given length (there is no validation that the current `cluster_name`s that you provided are the correct length).

Reply:
63 is the max of host name
54 is the max for cluster name

Reply:
so maybe it's worth to write a comment at the beginning of the test what is the max length for host & cluster name

Reply:
i actually would like my string to have - (dash) in between, and i cannot see other use than this test for a method to generate a string in certain length. WDYT about adding a method that will check\assert given string length?

Reply:
added assert_string_length